presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanNode.java

```diff
@@ -54,6 +58,11 @@ public PlanNodeId getId()
      */
     public abstract List<Symbol> getOutputSymbols();
 
+    public List<VariableReferenceExpression> getOutputVariables()
+    {
+        return ImmutableList.of();
```

Comment:
Can't do that because there are places calling this function while the PlaneNode hasn't really implemented it yet. That's why I introduced the validate function (if it's not implemented, the default is empty, so the validation checks it's either the same as symbol or it's empty).